internal/server/data/migrations_test.go

```diff
@@ -307,10 +341,28 @@ INSERT INTO provider_users (identity_id, provider_id, id, created_at, updated_at
 		}
 	}
 
+	var initialSchema string
 	for _, driver := range dbDrivers(t) {
 		if driver.Name() != "postgres" {
 			continue
 		}
+
+		t.Run("initial schema", func(t *testing.T) {
+			db := setupDB(t, driver)
+
+			initial, err := dumpSchema(os.Getenv("POSTGRESQL_CONNECTION"))
+			assert.NilError(t, err)
+			initialSchema = initial.String()
+
+			assert.NilError(t, db.Exec("DROP SCHEMA IF EXISTS testing CASCADE").Error)
+		})
+	}
+
+	for _, driver := range dbDrivers(t) {
+		if driver.Name() != "postgres" {
```

Comment:
Should we just load the postgres driver directly now instead?

Reply:
Ya, I think that would be better. I'll make that change